app/components/dds/dds-file-picker.js

```diff
@@ -27,10 +28,12 @@ const DDSFilePicker = Ember.Component.extend({
     if(! this.get('project.id')) {
       return;
     }
+    const fileFilter = this.get('fileFilter');
     this.get('store').query('dds-resource', {
       project_id: this.get('project.id')
     }).then((resources) => {
-      this.set('children', resources.sortBy('name'));
+      let filteredFiles = resources.filter(fileFilter);
+      this.set('children', filteredFiles.sortBy('name'));
```

Comment:
Hiding non-fastq files from the picker entirely may be misleading. I like the filter idea guided by the file format, but don't think we should hide other files entirely. Perhaps make them disabled from choosing - there's already mechanisms for disabling/graying out files that are already picked and this could be a simple computed property.